src/renderer/components/Navbar.vue

```diff
@@ -4,10 +4,17 @@
       <router-link to="/">♫ Polyphona</router-link>
     </h1>
     <ul>
-      <li>
+      <li v-if="user" id="display-username">
+        <small>Signed in as</small>
```

Comment:
J'aurais plutôt vu un symbole type :bust_in_silhouette: à la place de "signed in as"

Reply:
Ah bien vu ! C'est dans le style ;-)

Reply:
Voilou !